documentation/disaster-recovery.md

```diff
@@ -2,63 +2,99 @@
+- push the new branch to github
+- rerun the *enable maintenance workflow* using the new branch
+
 ### Recreate the lost postgres database server
 
 Run the deploy workflow to recreate the missing postgres database.
 
 As the maintenance page has been enabled, you will need to:
 - create a branch from main
-- update the env tfvars.json as per: [configure-terraform-to-keep-deploying-the-application](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/maintenance-page.md#configure-terraform-to-keep-deploying-the-application)
+- update the terraform application config as per: [configure-terraform-to-keep-deploying-the-application](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/maintenance-page.md#configure-terraform-to-keep-deploying-the-application)
 - push the branch to github
 - run the deploy workflow using your branch
 
 ### Restore the data from previous backup in Azure storage
 Run the *restore from backup* workflow.
 
 ### Validate app
-Confirm the app is working and is using the restored data.
+Confirm the app is working and can see the restored data. The app is available on the [temporary ingress](maintenance-page.md/#fail-over) URL.
 
 ### Disable maintenance mode
 Run the *disable maintenance workflow* for the service and environment affected.
 
+### Unfreeze pipeline
+
+Alert developers that merge to main is allowed.
+- In github settings, update the Branch protection rules and set required PR approvers back to 1
+
 ## Scenario 2: [Loss of data](https://technical-guidance.education.gov.uk/infrastructure/disaster-recovery/#data-corruption)
 
 In the case of data loss or corruption, we need to recover the data as soon as possible in order to resume normal service.
 
-The application's database is an Azure flexible postgres server. This provides a point-in-time restore (PTR) ability with the resolution of 1 second, available between 5min and 7days. PTR allows you to restore the source server to a point-in-time on a new copy of the server, not to the source server itself.
+The application database is an Azure flexible postgres server. This server has a point-in-time restore (PTR) ability with the resolution of 1 second, available between 5min and 7days. PTR allows you to restore the source server to a point-in-time on a new copy of the server. It does not update the source server itself in any way. Once the new server is available it can be accessed using [konduit.sh](https://github.com/DFE-Digital/teacher-services-cloud/blob/main/scripts/konduit.sh) to check previous data, and data can then be recovered to the original server.
 
 The objectives are:
 - Create a separate new postgres database server
 - Restore data from the current live database to the new postgres database server from a particular point in time
 - Update data into the source database from the new PTR server
 
+### Start the incident process (if not already in progress)
```

Comment:
Can this be moved after "Stop the service"? We should shut down early

Reply:
moved as requested